src/mip/coefficient_strengthening.jl

```diff
@@ -52,9 +52,9 @@ function coefficient_strengthening!(ps::PresolveData{T}, j::Int, inf::T = T(1e30
         urow = ps.urow[i]
 
         m = findfirst(isequal(j), row.nzind)
-        if m == nothing || (lrow > -inf && urow < inf)
+        if m == nothing || (lrow > -Inf && urow < Inf)
```

Comment:
just to ensure type-stability
```suggestion
        if m == nothing || (lrow > T(-Inf) && urow < T(Inf))
```